tests/test_bot.py

```diff
@@ -415,7 +414,8 @@ def test_delete_message_old_message(self, bot, chat_id):
     # duplicate here.
 
     @flaky(3, 1)
-    def test_send_venue(self, bot, chat_id):
+    @pytest.mark.parametrize('default_bot', [{'protect_content': True}], indirect=True)
+    def test_send_venue_with_default_protect_content(self, default_bot, chat_id):
```

Comment:
would you mind extracting the default-testing to a new test? it took a while for me to understand what's going on here and I think having two tests like for the other methods would make it easier